dlog/commitment/src/combine.rs

```diff
@@ -64,6 +65,36 @@ fn add_pairs_in_place<P: SWModelParameters>(pairs: &mut Vec<SWJAffine<P>>) {
     }
 }
 
+fn batch_add_assign_no_branch<P: SWModelParameters>(
```

Comment:
I highly suggest adding doc comments even for non-public functions. For example, I'm bit confused as what this function does here.